druntime/src/importc.h

```diff
@@ -33,6 +33,7 @@
 #define __attribute __attribute__
 #define __alignof _Alignof
 #define __vector_size__ vector_size
+#defome __typeof typeof
```

Comment:
```suggestion
#define __typeof typeof
```

Reply:
I would never make a misteak like that. It must be github corruption.